server/database.go

```diff
@@ -136,7 +136,7 @@ func databasePopulate() error {
 		}
 
 		// Skip non-music files
-		var extensions = [...]string{".mp3", ".ogg", ".flac", ".m4a"}
+		var extensions = [...]string{".flac", ".ogg", ".mp3"}
```

Comment:
Why is this reordered? Why has m4a been removed from the 'music extensions' list?

Reply:
It was re-ordered during my testing. I just kept it like that.

".m4a" was not there originally. I also added it in, then removed it again when I realized liquidsoap does not play m4a. (That is why I didn't add back much of CHVRCHES "Bones" album. My download was m4a. The Bones songs that are on Cadence now were converted from m4a -> mp3.)

Reply:
I feel like someone should file an issue against liquidsoap for that...

You could add ffmpeg as an external decoder to liquidsoap. Or see why faad isn't working...

https://www.liquidsoap.info/doc-dev/external_decoders.html